apps/extension/src/contents/inpage.ts

```diff
@@ -59,7 +62,7 @@ export const windowAI: WindowAI<ModelID> = {
   },
 
   async getCurrentModel() {
-    const requestId = _relayRequest(PortName.Model, {})
+    const requestId = _relayRequest(PortName.Model, undefined)
```

Comment:
nit: We can omit undefined here - any arg not supplied will be undefined by default.

Reply:
typescript actually throws an error! since the arg is required but may be undefined

Reply:
hmm - if it maybe undefined, I think we can make that arg optional (optional arg becomes undefined by default)